scripts/Dockerfile

```diff
@@ -60,5 +60,5 @@ RUN find /opt/arviz/ -type f -print0 | xargs -0 dos2unix
 RUN rm -f arviz/tests/saved_models/*.pkl
 RUN find -type d -name __pycache__ -exec rm -rf {} +
 
-#  Install editable using the setup.py
+# Install editable using the setup.py
 RUN /bin/bash -c "source /root/.bashrc" && pip install -e .
```

Comment:
Why do we need to install editable here each time the containe runs? We could do this earlier in the installation and it would still work?

Reply:
Because docker user should be able to change the source --> build docker quickly and run shell / tests.

And when it is editable mode, user can "dynamically" change src without cd to site-packages.

Reply:
Also it is not everytime container runs but at build step